tests/geometry.h

```diff
@@ -104,7 +104,6 @@ class Test {
 };
 
 pair<gtsam::NonlinearFactorGraph**, gtsam::Values**> load2D(string filename, Test** model, int maxID, bool addNoise, bool smart);
-pair<gtsam::NonlinearFactorGraph**, gtsam::Values**> load2D(string filename, gtsam::noiseModel::Diagonal** model, int maxID, bool addNoise, bool smart);
```

Comment:
why is this deleted

Reply:
It was causing all the indices in the matlab wrapper to be incremented, which was a mess. :confused: